Hai Reveal conversations: Rewrite A11 Sound The Alarm with fewer text walls and more player engagement (#7586)
endless-sky/endless-sky: data/hai/hai reveal 4 middle.txt

		event "hai and human ships at wormhole"
		log `Teeneep has been rejected by the Elders again, but has set in a motion a plan which has triggered an election instead. This snap election has closed the border and risked creating a diplomatic incident.`
		conversation
			`Upon your return, Teeneep sends an urgent demand for an audience with the Elders, along with a few paragraphs written in Hai. The meeting is called within an hour, and the room is entirely full. This may be the entire Elder Council; if so, they must consider this a grave matter. Teeneep and Sayari are waiting for you to follow them into the chambers.`
			`	The Elder moderating the proceedings calls Teeneep to testify, using your language. It seems they consider your participation critical.`

> 			`	Teeneep begins, "For the first time in three hundred years, we find ourselves beset on our borders by hostile forces. They have kidnapped five thousand of our people, and this council has been too paralyzed with fear of change to defend us. The Unfettered were once the defenders of all Hai, and I have convinced them to be so once again. Will you do what is right for our people, or will you be replaced?"`

Reviewer: ```suggestion
			`	Teeneep begins. "For the first time in three hundred years, we find ourselves beset on our borders by hostile forces. They have kidnapped five thousand of our people, and this council has been too paralyzed with fear of change to defend us. The Unfettered were once the defenders of all Hai, and I have convinced them to be so once again. Will you do what is right for our people, or will you be replaced?"`
```

Teeneep is threatening the council here, but she doesn't say explicitly what the council should do. Is it better to leave her implying that the Hai military should accompany the Unfettered, or make her say it outright? She doesn't say anything about it here. I might have missed something from an earlier mission that covers this for the player, though.

Author: The coordination with the Unfettered has to be much more than just a military escort. There's the matter of diplomacy with humans to prevent a war, and internal politics to get the Fettered Hai populace to accept it. I thought it would be best to avoid such a long discussion. Presumably, the council wouldn't get into such details unless they thought an agreement to work with Unfettered is likely. As such an agreement is almost unimaginable, I'd expect them to skip the talk unless the vote went in a deeply surprising direction.

Author: I'm hoping more people will review, so I'll wait to hear more before I make such a large change to the conversation.

Author: I remember now the original justification for the Unfettered help. The Fettered are beset on both sides by attackers, but Teeneep wants a deal where the Unfettered stop attacking the Fettered and guard the wormhole. 

That doesn't work anymore, now that the cease in intra-Hai hostilities is gone. I'll have to contrive another justification.

Author: I think the best solution is for Teeneep to demand the Elders allow the Unfettered to pass through Fettered territory and assist in defending the wormhole. Ultimately, that is what ends up happening, but only a few Unfettered special ops ships rather than a full alliance.

Author: Teeneep now insists on an agreement with the Unfettered that is also negotiated with the humans. That way, the player's continued cooperation isn't suspicious (as they may be intentionally starting a war otherwise). I've updated the player's possible responses to match the new Teeneep speech.

			`	At the word "replaced," murmurs rise and Sayari looks at you with deep concern. The moderator silences the room and asks you to testify. "Captain <last>, you have the unique vantage point of an accomplished outsider who has seen all parts of this conflict. What do you advise?`

> 			`	You carefully choose your words before speaking, as this may be a pivotal moment in history. This may spark a war - or prevent one.`

Reviewer: ```suggestion
			`	You should carefully choose your words before speaking, as this may be a pivotal moment in history. This may spark a war - or prevent one.`
```

			choice

> 				`	"You can trust the Unfettered to defend you, but you must be careful to avoid a war with the humans. That would be catastrophic for everyone."`

Reviewer 2: I really like that they're asking your opinion for the UHai
Mb they could ask for the kidnapping threat as well? Since you're most expert in that, and actually saw victims first-hand

Idk why FL would be an UHai expert more than they are honestly, but kidnapping is where the expertise is at (though doesnt hurt to ask for UHai sincerity)

Author: I meant for the moderator's question to imply that he's asking about the full situation, including the kidnappings and the tensions with human governments. Perhaps I need to carefully reword his question.

Reviewer 2: I just feel like thats where they should ask your opinion mostly

Author: Yes, that makes sense. I'll have them ask that before they ask about the Unfettered decision.

Author: I added a question to the player about the kidnappings.

				`	"The Unfettered have offered their help because you would not help your people. If you change your decision, you will not need their involvement."`
					goto "turn back unfettered"
			action
				set "told hai to accept unfettered help"
			`	One Elder rises and says, "The Unfettered only know how to make war. Once they are in human space, there's no telling what they'll do." Many in the room nod in agreement.`
			`	Teeneep responds, "They are clever strategists and dedicated in our protection. Never would they take such unnecessary risks. I am certain they would accept an overseer among us to advise them on limits to their actions."`
				goto "the elders vote"
			label "turn back unfettered"
			action
				set "told hai to reject unfettered help"
			`	A disgruntled Elder responds, "Change our decision? Our decision not to violate human sovereignty? Even if I lose my Eldership, I will not vote for violence."`

> 			`	Teeneep says, "If you take no action, and refuse help from our brethren, they'll pass through our territory anyway. Fighting at Wah Ki has stopped, as the military is anticipating an agreement."`

Reviewer 2: I think that if fighting stopped at wah ki we need to make it actually stop trough an event

Author: Can you please find where the code is that stops the fighting?

I don't know where it happens, but it isn't in this mission. It's earlier.

Reviewer 2: Oh so it actually happens? Because the old text said it had intensified or smth?

Author: It is possible I removed it by accident. I'll replay the Hai Reveal and make sure it's still there.

Reviewer 2: I played trough HR I'm pretty sure it did not happen

Author: It definitely should happen, as the rest of Hai Reveal doesn't make sense without it.

Author: I don't want to touch that in this PR though. Filling that plot hole would require a code change in the Wanderers campaign too. Instead, I'll just remove Teeneep's statement.

Reviewer 2: Idk if it would require changes in Wand honestly
Yeah also they say it intensifies since the start but it didnt change one bit bruh

Could be for another PR for sure

Author: The change in the Wanderers campaign is to restore the hostility. In other words, the Hai Reveal would change the affinity so the Hai factions are friends, but Sayari's visit will change them back to enemies.

That definitely does *not* belong in this PR though, so I'll just modify the dialog so it doesn't need a new government affinity.

Author: After re-reading the Hai Reveal/Wanderers campaign connections, I see some bigger plot holes. Somehow the enormous political clout Teeneep had mysteriously never existed in the Wanderers campaign. She also seems to be hugely pro-Unfettered, not caring much about the Wanderers, even though she put a lot of work into keeping peace with the humans during their dealings with the Unfettered. This is easy to fix without substantive changes to the Wanderers campaign, but I'll work on that after 7586 has more review.

Reviewer 2: > The change in the Wanderers campaign is to restore the hostility. In other words, the Hai Reveal would change the affinity so the Hai factions are friends, but Sayari's visit will change them back to enemies.
> 
> That definitely does _not_ belong in this PR though, so I'll just modify the dialog so it doesn't need a new government affinity.

Oh it makes them friendly I thought it stopped the incoming ships
I saw a new gov "friendly uhai" was made so I thought that wasnt the case

Author: No, that's only for the ships the Hai allow to pass through their territory. I'm 83% certain all other Unfettered fleets are in the original government.

Author: Teeneeps prior mention of Unfettered not attacking is now replaced with a warning: the Fettered probably could not stop the entire Unfettered armada, and it is unnecessary bloodshed anyway.

			label "the elders vote"
			`	Another Elder interjects, "You're exaggerating the impact, Teeneep. Five thousand Hai in two hundred years is barely a concern. That many managed to get themselves killed swimming with Gelo Appanoh in the last decade alone."`
			`	Many Elders seem appalled by that observation, but the moderator interrupts. "We will vote on two counts: Do we act? Do we accept Unfettered help?"`
			choice
				`	(Await the vote.)`
			`	After a few more minutes of private discussions, the vote is tallied. Unfortunately, the majority vote "no" on both counts. Your words may not have swayed the vote, but perhaps they'll resonate in the Hai psyche long enough to have an effect.`
			`	Teeneep is allowed to respond, and says ominously, "You know what will happen next."`

> 			`	An angry Elder replies, "You wouldn't dare!" To which Teeneep responds, "It is already scheduled to happen unless I actively stop it!" She smiles and leaves the room as Sayari follows, motioning for you to do so as well.`

Reviewer: ```suggestion
			`	An angry Elder replies, "You wouldn't dare!" to which Teeneep responds, "It is already scheduled to happen unless I actively stop it!" She smiles, a little self indulgently, and withdraws from the room. Sayari follows, motioning for you to do so as well.`
```

			`	Sayari leads you to a private meeting room.`
			`	Teeneep explains her ominous words, "Our democracy allows a Petition of No Confidence against the government. If there are enough petitioners, it forces an election. Activists in my organization have been pushing a petition for weeks.`
			`	"It was obvious what the Elder Council's response would be," and with a sarcastic smile, she adds, "and I knew I would only come back alive if the Unfettered agreed to help. So, I arranged in advance for the petition to be promoted along with a widespread broadcast of these proceedings."`
			`	Sayari assures you, "These petitions take time, and rarely succeed. It is more of a warning to the Elders than anything else. Still, we should watch the situation.`

> 			`	"For now," she says, "we're all in need of lunch"`

Reviewer: ```suggestion
			`	"For now," she says, "we're all in need of some lunch."`
```

			choice

> 				`	(Go to lunch.)`

Reviewer 2: just wondering why did you remove the option to have lunch with diplomats?

Author: It causes you to miss critical plot points.

Author: That's in the PR description, "Removed the option to eat with Alondo since it skipped critical information."

		conversation
			`Upon your return, Teeneep sends an urgent demand for an audience with the Elders, along with a few paragraphs written in Hai. The meeting is called within an hour, and the room is entirely full. This may be the entire Elder Council; if so, they must consider this a grave matter. Teeneep and Sayari are waiting for you to follow them into the chambers.`
			`	The Elder moderating the proceedings calls Teeneep to testify, using your language. It seems they consider your participation critical.`
			`	Teeneep begins. "For the first time in three hundred years, we find ourselves beset on our borders by hostile forces. They have kidnapped five thousand of our people, and this council has been too paralyzed with fear of change to defend us. The Unfettered were once the defenders of all Hai, and I have convinced them to be so once again. Will you do what is right for our people, or will you be replaced?"`
			`	At the word "replaced," murmurs rise and Sayari looks at you with deep concern. The moderator silences the room and asks you to testify.`
			`	"Captain <last>, can you speak to the truth of these disappearances?"`
			choice

> 				`	"There evidence is circumstantial put compelling, and we must expand the investigation."`

Reviewer: ```suggestion
				`	"There evidence is circumstantial but compelling. We must expand the investigation."`
```

> 				`	"The evidence that they are kidnappings is circumstantial but compelling. We must expand the investigation."`

Reviewer: ```suggestion
				`	"The evidence that there are ongoing kidnappings is circumstantial, but compelling. We must expand the investigation."`
```
Whoops, messed this one up last time.

					goto uncertainties
				`	"It is a coordinated kidnapping campaign, and we must put a stop to it."`
			`	You hear many murmurs of dissent, but the moderator raises his paws to delay discussion.`
				goto "next question"
			label uncertainties
			`	Teeneep looks mildly perturbed, while many Elders nod with agreement.`
			label "next question"

> 			`	The moderator looks deeply serious, says, "Captain <last>, you have the unique vantage point of an accomplished outsider who has seen all parts of this conflict. We have become unwittingly embroiled in human internal conflicts, and attacked for the first time in 300 years. Much of this is beyond our experience, but perhaps not beyond yours. What do you advise?"`

Reviewer: ```suggestion
			`	The moderator looks deeply serious. He says, "Captain <last>, you have the unique vantage point of an accomplished outsider who has seen all parts of this conflict. We have become unwittingly embroiled in human internal conflicts, and have been attacked for the first time in 300 years. Much of this is beyond our experience, but perhaps not beyond yours. What do you advise?"`
```

			`	A disgruntled Elder responds, "Change our decision? Our decision not to violate human sovereignty? Even if I lose my Eldership, I will not vote for violence."`
			`	Teeneep says, "If you take no action, and refuse help from our brethren, they'll pass through our territory anyway. We can hold back Unfettered raids, but their entire armada? That is the unnecessary violence you bring through inaction."`
			label "the elders vote"
			`	Another Elder interjects, "You're exaggerating the impact, Teeneep. Five thousand Hai in two hundred years is barely a concern. That many managed to get themselves killed swimming with Gelo Appanoh in the last decade alone."`
			`	Many Elders seem appalled by that observation, but the moderator interrupts. "We will vote on two counts: Do we act? Do we accept Unfettered help?"`
			choice
				`	(Await the vote.)`
			`	After a few more minutes of private discussions, the vote is tallied. Unfortunately, the majority vote "no" on both counts. Your words may not have swayed the vote, but perhaps they'll resonate in the Hai psyche long enough to have an effect.`
			`	Teeneep is allowed to respond, and says ominously, "You know what will happen next."`

> 			`	An angry Elder replies, "You wouldn't dare!" to which Teeneep responds, "It is already scheduled to happen unless I actively stop it!" She smiles, a little self indulgently, and withdraws from the room. Sayari follows, motioning for you to do so as well.`

Reviewer: ```suggestion
			`	An angry Elder replies, "You wouldn't dare!" to which Teeneep yells back, "It is already scheduled to happen unless I actively stop it!" She smiles, a little self indulgently, and withdraws from the room. Sayari follows, motioning for you to do so as well.`
```
Now I think 'responds' is a little dull, so I'm going to change it again to 'yells back'.

			`	Sayari leads you to a private meeting room.`

> 			`	Teeneep explains her ominous words, "Our democracy allows a Petition of No Confidence against the government. If there are enough petitioners, it forces an election. Activists in my organization have been pushing a petition for weeks.`

Reviewer: ```suggestion
			`	Teeneep explains her ominous words. "Our democracy allows a Petition of No Confidence against the Elders. If there are enough petitioners, it forces an election. Activists in my organization have been pushing a petition for weeks.`
```

> 			`	"It was obvious what the Elder Council's response would be," and with a sarcastic smile, she adds, "and I knew I would only come back alive if the Unfettered agreed to help. So, I arranged in advance for the petition to be promoted along with a widespread broadcast of these proceedings."`

Reviewer: ```suggestion
			`	"It was obvious what the Elder Council's response would be," and with a sarcastic smile she adds, "and I knew I would only come back alive if the Unfettered agreed to help. So, I arranged in advance for the petition to be promoted along with a widespread broadcast of these proceedings."`
```
I'd like to try get rid of this repetition with the smiling.

Author: I changed it to "with a bit of sarcasm"

			`	At the word "replaced," murmurs rise, and Sayari looks at you with deep concern. The moderator silences the room and asks you to testify.`
			`	"Captain <last>, can you speak to the truth of these disappearances?"`
			choice
				`	"The evidence that there are ongoing kidnappings is circumstantial, but compelling. We must expand the investigation."`
					goto uncertainties
				`	"It is a coordinated kidnapping campaign, and we must put a stop to it but take care not to start a war."`
			`	You hear many murmurs of dissent, but the moderator raises his paws to delay discussion.`
				goto "next question"
			label uncertainties
			`	Teeneep looks mildly perturbed, while many Elders nod with agreement.`
			label "next question"

> 			`	The moderator turns back to you with a deeply serious face. "Captain <last>, you have the unique vantage point of an accomplished outsider who has seen all parts of this conflict. We have become unwittingly embroiled in human internal conflicts and have been attacked for the first time in 300 years. Much of this is beyond our experience, but perhaps not beyond yours. What do you advise?"`

Reviewer: ```suggestion
			`	The moderator turns back to you with a highly serious face. "Captain <last>, you have the unique vantage point of an accomplished outsider who has seen all parts of this conflict. We have become unwittingly embroiled in human internal conflicts and have been attacked for the first time in 300 years. Much of this is beyond our experience, but perhaps not beyond yours. What do you advise?"`
```
Get rid of some repetition that could happen.

			`	Teeneep responds, "They are clever strategists and dedicated in our protection. Never would they take such unnecessary risks. I am certain they would accept an overseer among us to advise them on limits to their actions."`
				goto "the elders vote"
			label "turn back unfettered"
			action
				set "told hai to reject unfettered help"
			`	A disgruntled Elder responds, "Change our decision? Our decision not to violate human sovereignty? Even if I lose my Eldership, I will not vote for violence."`
			`	Teeneep says, "If you take no action, and refuse help from our brethren, they'll pass through our territory anyway. We can hold back Unfettered raids, but their entire armada? That is the unnecessary violence you bring through inaction."`
			label "the elders vote"
			`	Another Elder interjects, "You're exaggerating the impact, Teeneep. Five thousand Hai in two hundred years is barely a concern. That many managed to get themselves killed swimming with Gelo Appanoh in the last decade alone."`
			`	Many Elders seem appalled by that observation, but the moderator interrupts. "We will vote on two counts: Do we act? Do we accept Unfettered help?"`
			choice

> 				`	"Teeneep, what happens next?"`

Reviewer: ```suggestion
				`	"What's going to happen, Teeneep?"`
```

				goto frantic
			label spicy
			`	Sayari takes you to her favorite restaurant, a famous Korean-Hai fusion place. She tells the waiter, "I won't leave happy unless my mouth is on fire." You try a bite from Sayari's plate, but can't swallow without gagging. She seems to enjoy the inferno of capsaicin.`
				goto frantic
			label pizza
			`	You join Teeneep and Sayari at a fusion restaurant about a block down the road from the "Elders' Chambers" - which turns out to be the official name of the whole government complex. The pizza is as good as the rumors you've heard, even though the "cheese" is processed nut butter.`
			label frantic
			`	Not half way through her food, Sayari's data pad beeps frantically. She looks at it and mutters, "Oh dear, oh no." She lapses into Hai for a moment before returning. "The petition! It hit the trending issues board and triggered the safety policy alert response. People got direct notifications that this was an issue. There are over two billion supporting votes for an injunctive action, and it's only been an hour... this is what you hoped would happen, isn't it?"`
			`	Teeneep places a gentle hand on Sayari's arm. "Better this than war."`

> 			`	Sayari's eyes are alive with thinly-veiled panic as she slams her hand down on the table. Teeneep seems shocked, as if she had never seen Sayari so emotional.`

Reviewer: ```suggestion
			`	Sayari's eyes are alive with thinly-veiled panic as she slams her paw down on the table. Teeneep seems shocked, as if she had never seen Sayari so emotional.`
```
I don't like using paw instead of hand, but it's more consistent with everything else.

		event "hai and human ships at wormhole"
		log `Teeneep has been rejected by the Elders again, but has set in a motion a plan which has triggered an election instead. This snap election has closed the border and risked creating a diplomatic incident.`
		conversation
			`Upon your return, Teeneep sends an urgent demand for an audience with the Elders, along with a few paragraphs written in Hai. The meeting is called within an hour, and the room is entirely full. This may be the entire Elder Council; if so, they must consider this a grave matter. Teeneep and Sayari are waiting for you to follow them into the chambers.`
			`	The Elder moderating the proceedings calls Teeneep to testify, using your language. It seems they consider your participation critical.`

> 			`	Teeneep begins. "For the first time in three hundred years, we find ourselves beset on our borders by hostile forces. They have kidnapped five thousand of our people, and this council has been too paralyzed with fear of change to defend us. The Unfettered were once the defenders of all Hai, and I have convinced them to be so once again. They are ready to send a fleet to human space and extinguish the pirate threat. Will you do what is right for our people and negotiate this with the humans, or will you be replaced?"`

Reviewer 3: I went to make three different suggestions but none of them were right.

This opening sentence has the following problems:

- It was less than 300 years (also we're trying not to be too specific about that because the date the player gets to this varies).
- If this issue is ongoing/cyclical then it's not the first time, and describing it as such weakens her position.
- We have not necessarily established why the council is inactive, or at least not presented the possible motivations (I don't think - feel free to correct me) which makes this accusation incredibly rude.
- Also, it's bad politic to directly insult the people you wish to galvanise to action when you've already indirectly insulted them by working behind their backs. This current presentation implies she *wants* them to refuse her...
- The Unfettered never stopped wanting to be defenders of all Hai, the only thing they've been convinced of is a willingness to properly fight the Hai over the moral issue of inaction.
- Also, most Unfettered alive were probably not in fact the actual defenders. They'd be a minority at this stage.
- Extinguishing the pirate threat entirely is not really possible without mass conducting atrocities, which the Elders know.
- What they've already been doing could well be described as "negotiating with humans", what's being asked is for them to demand action from specific humans (The Republic).

These Elders have been politicking for a long time. I wouldn't imagine that Teeneep would get to the end of this paragraph successfully without being interrupted a half dozen times and likely dismissed by half those assembled.

			`	At the word "replaced," murmurs rise, and Sayari looks at you with deep concern. The moderator silences the room and asks you to testify.`
			`	"Captain <last>, can you speak to the truth of these disappearances?"`
			choice
				`	"The evidence that there are ongoing kidnappings is circumstantial, but compelling. We must expand the investigation."`
					goto uncertainties

> 				`	"It is a coordinated kidnapping campaign, and we must put a stop to it but take care not to start a war."`

Reviewer 3: These are both comparatively a lot of words for <first> to say in a choice.
They should be re-assessed when the preceding text changes.

			`	You hear many murmurs of dissent, but the moderator raises his paws to delay discussion.`
				goto "next question"
			label uncertainties
			`	Teeneep looks mildly perturbed, while many Elders nod with agreement.`
			label "next question"

> 			`	The moderator turns back to you with a highly serious face. "Captain <last>, you have the unique vantage point of an accomplished outsider who has seen all parts of this conflict. We have become unwittingly embroiled in human internal conflicts and have been attacked for the first time in 300 years. Much of this is beyond our experience, but perhaps not beyond yours. What do you advise?"`

Reviewer 3: Again, the 300 years.
However, in this context it is worth considering that the player is not a politician, their character has no diplomatic or policy backgrounding.
If some subset of the Elders are actually engaged enough to want to address possible options here, it seems to me far more likely that they would speak of a selection of options that they've already been given or requested from various people (or have devised themselves) and query you on which one you think is most reasonable/problematic. (As an outsider you can't be expected to propose a solution, but *can* be expected to spot a problem that others might have overlooked.)

I would expect that the options they'd present would looks something like:
- Their favourite
- The one they think is most practical
- The one they think is most ridiculous

Not necessarily in that order, and not necessarily with any indication of which is which.

					goto teeneep
				`	"Sayari, what happens next?"`
			`	Sayari explains, "There's a multi-step voting process, which takes into account dissent between Elders. After each stage of voting, there's discussions between strategically chosen groups of Elders, and afterwards the dissent is usually lower. They iterate until they come to a consensus, unless it takes too long; then they'll use the majority vote."`
				goto results
			label teeneep
			`	Teeneep responds, "They'll vote to do nothing. It's all this council knows how to do." Sayari sighs loudly at this but doesn't contradict Teeneep.`
			label results
			`	After a few more minutes of private discussions, the Elders vote "no" on both counts. Your words may not have swayed the vote, but perhaps they'll resonate in the Hai psyche long enough to have an effect.`
			`	Teeneep is allowed to respond and says ominously, "You know what will happen next."`
			`	An angry Elder replies, "You wouldn't dare!" Teeneep yells back, "It is already scheduled to happen unless I actively stop it!" She smiles, a little self indulgently, and withdraws from the room. Sayari follows, motioning for you to do so as well.`

> 			`	Sayari leads you to a private meeting room.`

Reviewer 3: This sequence is gratuitous and feels forced now.

Author: I left this awkward part in because an Elder is supposed to threaten Teeneep. I will come up with a different way for the Elder to threaten.

			`	Teeneep explains her ominous words. "Our democracy allows a Petition of No Confidence against the Elders. If there are enough petitioners, it forces an election. Activists in the organization I founded have led a grassroots movement pushing a petition for weeks.`
			`	"It was obvious what the Elder Council's response would be," and with a bit of sarcasm she adds, "and I knew I would only come back alive if the Unfettered agreed to help. So, my supporters arranged in advance for the petition to be promoted along with a widespread broadcast of these proceedings."`

> 			`	Sayari assures you, "These petitions take time and rarely succeed. It is more of a warning to the Elders than anything else. Still, we should watch the situation.`

Reviewer 3: One of the objectives of the original piece was to show the Hai as a politically engaged people, but a people that are undecided.
A people equipped with a mechanism for making direct and radical change, but a predilection for disagreeing on thing sufficiently that it rarely amounted to being more than an opinion poll for the Elder's to observe the results of.

This also has issues with being presented as a grassroots movement:

- Such movements tend to take more than a few weeks.
- The Hai population is not small, to push a movement like that quickly would require Teeneep's organisation to be *very* large. (Which is why originally she was leveraging existing infrastructure, with the aid of her organisation's influence.)
- You know the whole bit about needing to confront the Elders directly and conform to the process? That is *directly* at odds with the concept of making sensitive information publicly available in the background as part of some social campaign.
- Corollary: if that campaign were successful enough to already be at this stage, the Elders would have heard about it already and she'd have been summarily censured for breach of process and betrayal of trust.

				goto frantic
			label spicy
			`	Sayari takes you to her favorite restaurant, a famous Korean-Hai fusion place. She tells the waiter, "I won't leave happy unless my mouth is on fire." You try a bite from Sayari's plate, but can't swallow without gagging. She seems to enjoy the inferno of capsaicin.`
				goto frantic
			label pizza
			`	You join Teeneep and Sayari at a fusion restaurant about a block down the road from the "Elders' Chambers" - which turns out to be the official name of the whole government complex. The pizza is as good as the rumors you've heard, even though the "cheese" is processed nut butter.`
			label frantic
			`	Not half way through her food, Sayari's data pad beeps frantically. She looks at it and mutters, "Oh dear, oh no." She lapses into Hai for a moment before returning. "The petition! It hit the trending issues board and triggered the safety policy alert response. People got direct notifications that this was an issue. There are over two billion supporting votes for an injunctive action, and it's only been an hour... this is what you hoped would happen, isn't it?"`
			`	Teeneep places a gentle hand on Sayari's arm. "Better this than war."`

> 			`	Sayari's eyes are alive with thinly-veiled panic as she slams her paw down on the table. Teeneep seems shocked, as if she had never seen Sayari so emotional.`
> 			choice
> 				`	"What's wrong, Sayari?"`
> 			`	She says frantically, "Captain <last>, you must leave immediately!`

Reviewer 3: Everything about this sequence felt correct but off in how it was presented.
```suggestion
			`	Sayari pauses; staring at her friend for a long moment before breaking contact. She draws a surprised gasp from Teeneep as she suddenly slams her paw down on the table and turns to you, her eyes wide with thinly-veiled panic.`
			`	"<last>, you must leave immediately!" she says, frantically.`
			choice
				`	"What's wrong?"`
```
